study.md

```diff
@@ -29,21 +29,25 @@ Asynchronous - operations that may start in a certain sequence and may end in an
 In your own words, describe the event loop and how it works.
 
 ```md
-<!-- your answer here -->
+The event loop is the order which Javascript is executed and helps keep track of what to run next.
```

Comment:
So the event loop is what handles asynchronous code. The stack is the order of execution of synchronous code, when some async code needs to run it waits in the queue. The event loop waits until the stack is empty, then it pushes the code from the queue to the stack so it can run.